README.md

```diff
@@ -1,3 +1,87 @@
-# Certifly
+# Certifly - Web-based Certificate Generation Tool
 
-Certificate generator
+Certifly is an open-source web-based certificate generation tool written in Svelte by [Kathmandu university Open Source Commmunity (KUOSC)](https://kucc.ku.edu.np/kuosc/). It simplifies the process of creating certificates by allowing users to provide a template image and a CSV file containing recipient information.
+
+## Table of Contents
+
+- [Features](#features)
+- [Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Installation](#installation)
+- [Usage](#usage)
+- [Contributing](#contributing)
+- [License](#license)
+
+## Features
+
+- **Easy Certificate Generation**: Generate customized certificates by providing a template image and recipient data in CSV format.
+
+- **Template Customization**: Customize certificate appearance using template images with placeholders.
+
+- **Supports Various Formats**: Certifly supports PNG and JPEG/JPG image formats for templates and CSV data.
+
+- **Bulk Export**: Export generated certificates in bulk as a zip file.
+
+## Getting Started
+
+### Prerequisites
+
+Before you begin, ensure you have the following installed:
+
+- [Node.js](https://nodejs.org/)
+- [pnpm](https://pnpm.io/) (Performant NPM)
+
+### Installation
+
+1. Clone the Certifly repository to your local machine:
+
+   ```bash
+   git clone git@github.com:kuosc2005/certifly.git
+   cd certifly
+   ```
+
+2. Install the project dependencies:
+
+   ```bash
+   pnpm install
+   ```
+
+## Usage
+
+1. **Prepare your template image:**
+
+   Create a template image (e.g., `certificate_template.png`) with placeholders where you want recipient information to appear.
```

Comment:
you don't add template image with placeholders, you click on the template image, and then the new field gets added.